addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -969,23 +1081,38 @@
             <field name="children_tax_ids" eval="[(6,0,[ref('tax_ust_7_purchase_13a_auslagerung_skr03'), ref('tax_vst_7_purchase_13a_auslagerung_skr03')])]"/>
             <field name="tax_group_id" ref="tax_group_0"/>
         </record>
+        <record id="fiscal_position_domestic_skr03" model="account.fiscal.position.template">
+            <field name="sequence">1</field>
+            <field name="name">Geschäftspartner Inland</field>
+            <field name="chart_template_id" ref="l10n_de_chart_template"/>
+            <field name="auto_apply" eval="True" />
+            <field name="country_id" ref="base.de"></field>
+        </record>
         <record id="fiscal_position_non_eu_partner_service_skr03" model="account.fiscal.position.template">
+            <field name="sequence">6</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
-            <field name="name">Dienstleister Ausland</field>
+            <field name="name">Dienstleister Ausland (Nicht-EU)</field>
         </record>
         <record id="fiscal_position_non_eu_partner_skr03" model="account.fiscal.position.template">
+            <field name="sequence">5</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
-            <field name="name">Geschäftspartner Ausland</field>
+            <field name="name">Geschäftspartner Ausland (Nicht-EU)</field>
         </record>
         <record id="fiscal_position_eu_vat_id_partner_skr03" model="account.fiscal.position.template">
+            <field name="sequence">2</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
             <field name="name">Geschäftspartner EU (mit USt-ID)</field>
```

Comment:
Is not this also vat_required?

Reply:
@jco-odoo Theoretically you are right, however to my opinion it is error prone in the whole context of all fiscal positions. 

Imagine we would have a partner where we would add a country from the EU, which is the first logic and simple step for a user. To my experience users doesn't search and assign a VAT-ID in Odoo, even if there is definitely a VAT-ID existing for that partner.
 
For such cases the consequence in the context of all our fiscal positions would be that we could not auto-apply any fiscal position even if a user would have defined the country for a partner (from EU country). To my opinion we would apply a user trap by adding vat_required to that very important fiscal position which covers all B2B tax cases with customers / suppliers from the EU just by adding the country. From usability point of view we shouldn't be so strict at that point.

Reply:
@jco-odoo For sure i should also remark that for a B2C case for a partner from the EU (partner really doesn't have a VAT-ID) it would be required to add the right fiscal position (sequence no "4") by hand to get the right taxes.